Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -1955,7 +1955,9 @@ BYE BYE BORDERS
         cursor: default;
         background-color:  #f4f4f4;
     }
-    .z-cse #contentPaneContainer input.tableheader:hover {
+    /*ZEN-30221*/
+    .z-cse #contentPaneContainer input.tableheader:hover,
+    .z-cse.z-cse-dark #contentPaneContainer input.tableheader:hover {
```

Comment:
Do we need the same selector for `.z-cse.z-cse-dark`? 
If so please move it under the "Dark theme" comment.

Reply:
done